src/components/App.js

```diff
@@ -1,22 +1,17 @@
 import React from 'react'
-import { Root, Container, Spacer } from './Common'
+import { Root, Container } from './Common'
 import Router from './Router'
-import Header from './Header'
 
 export default function App() {
   return (
     <Root display={'flex'} justifyContent={'center'}>
       <Container
-        size={10}
         styles={{
-          flexDirection: 'column',
-          maxHeight: '1024px',
-          maxWidth: '1366px',
+          maxHeight: '1080px',
```

Comment:
The screen size I've taken from figma. Not sure if this was updated or not.

Reply:
We were told that the resolution was going to be 1920x1080, so Jon set the background to that size